src/main/java/mate/academy/service/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://127.0.0.1:3306/test?user=root";
+    private static final Properties DB_PROPERTIES;
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "Nero0410";
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", USERNAME);
+        DB_PROPERTIES.put("password", PASSWORD);
+
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can not load JDBC driver", e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
```

Comment:
It's good practice to use try-with-resources or closing the connection explicitly in the calling code to prevent potential resource leaks. Ensure that connections are closed properly after use.